source/includes/_invoice-payment.md

```diff
@@ -383,7 +406,21 @@ curl -v \
 ```
 
 ```java
-TODO
+import org.killbill.billing.client.api.gen.InvoicePaymentApi;
+import org.killbill.billing.client.model.gen.InvoicePaymentTransaction;
+
+protected InvoicePaymentApi invoicePaymentApi;
+
+
+UUID payment_id = UUID.fromString("b9976ff0-d326-42bd-8e91-ea083149a38f");
+String chargeback_transaction_external_key = "99c45d07-abe4-4bc7-a207-0524548c1b08";
```

Comment:
consider renaming to `chargebackTransactionExternalKey` as per Java coding conventions.